test/collection.test.js

```diff
@@ -8,15 +8,18 @@ const assert = require('assert');
 const mongoose = start.mongoose;
 
 describe('collections:', function() {
-  const connectionsToClose = [];
+  let db = null;
 
-  after(async function() {
-    await Promise.all(connectionsToClose.map((v) => v.close()));
+  this.afterEach(async function() {
```

Comment:
```suggestion
  afterEach(async function() {
```

i dont think `this.` is required here, removing it to make it consistent with other test files